packages/web-app-files/src/views/trash/Overview.vue

```diff
@@ -175,7 +174,27 @@ export default defineComponent({
     ])
 
     const getSpaceName = (space: SpaceResource) => {
-      return isPersonalSpaceResource(space) ? $gettext('Personal') : space.name
+      if (isPersonalSpaceResource(space)) {
+        return $gettext('Personal')
+      }
+      if (space.disabled === true) {
+        return $gettext('%{spaceName} (disabled)', { spaceName: space.name })
+      }
+
+      return space.name
+    }
+
+    const getSpaceAttributes = (space: SpaceResource) => {
+      if (isProjectSpaceResource(space) && space.disabled !== false) {
```

Comment:
```suggestion
      if (isProjectSpaceResource(space) && space.disabled === true) {
```
oh sorry, here as well